forms-flow-web/src/components/Form/List.js

```diff
@@ -219,7 +219,8 @@ const List = React.memo((props) => {
           });
         }
         else {
-          dispatch(push(`${redirectUrl}formflow/${form._id}/edit/`));
+          let form_id = res?.data?.formId;
+          dispatch(push(`${redirectUrl}formflow/${form_id}/edit/`));
```

Comment:
can u optimise in this way so that dispatch happens only if value present @Josephalexantony-aot 

res?.data?.formId && dispatch(push(`${redirectUrl}formflow/${res.data.formId}/edit/`));

Reply:
sure